languages/en.json

```diff
@@ -32,6 +32,15 @@
         "error": "Please try again. If this issue persists, join my Discord server to report a bug.",
         "website": "My Website",
         "button": "Module Compatibility Checker",
-        "moduleManagementColorsExplanation": "The background colors below are used to indicate the status of the module in the latest core version. Open the Module Compatibility Checker with the button below for more info."
+        "moduleManagementColorsExplanation": "The background colors below are used to indicate the status of the module in the latest core version. Open the Module Compatibility Checker with the button below for more info.",
+			  "scrollToTop": "Scroll To Top",
+			  "detailsHide": "Hide details",
+			  "detailsShow": "Show details",
+			  "percentageFilteredTooltip": "The percentage ready for current filtered package count.",
+			  "percentageKnownTooltip": "The percentage of all \"known\" ({known}) compatible packages with current spreadsheet data is: {percentageKnown}%.",
+				"linkFoundryPackage": "Foundry Package Listing (May not exist)",
+				"linkPackageUrl": "Package URL / Repository",
+				"showKnownPackages": "Show currently known compatible packages.",
+				"resetAllFilters": "Reset all filters."
```

Comment:
The indentation is incorrect here

Reply:
Don't know what to tell ya.... If it looks wrong on your side... Easy fix: 

![image](https://user-images.githubusercontent.com/311473/179153697-ce308231-2915-4f39-8aab-e4adc57e9fb8.png)